src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -68,7 +68,6 @@
 import org.apache.jena.rdf.model.Statement;
 import org.apache.jena.rdf.model.StmtIterator;
 import org.apache.jena.riot.RDFDataMgr;
-
```

Comment:
Needless formatting changes like this are often a source of merge conflicts. Please remove.